tests/Features/homepage.feature

```diff
@@ -0,0 +1,6 @@
+Feature: Homepage
+
+    Scenario: It loads!
+        Given I am on the homepage
+        Then I should see "OpenCFP"
+          And I should not see "Internal Server Error"
```

Comment:
Is there a way to automatically check this, and maybe for other errors as well, on every page visit?

Reply:
I would stick with positive assertions. If you get an ISE, it's very likely a step definition for the affected scenario will fail. Additionally, we shouldn't check for such things in a "Homepage" feature (which I may remove instead).

I think adding an "Error Handling" feature that covers things like our 404, 500 handlers would be reasonable; especially if we're going to be improving those (e.g.: 500 error because "you forgot to configure the app" versus "uncaught exception", etc)